lib/tests/streamlit/scriptrunner/ScriptRunner_test.py

```diff
@@ -457,21 +456,41 @@ def get_widget_id(self, widget_type, label):
         return None
 
 
-def wait_for_widgets_script_deltas(runners):
-    """Wait for the 9 deltas produces by the first loop of widgets_script.py.
-
-    This is meant to deflake tests that previously depended on timing to ensure
-    we had all the deltas we needed.
+def require_widgets_deltas(
+    runners: List[TestScriptRunner], timeout: float = 15
```

Comment:
Remove the timeout argument since it's never used. Just set it internally to 15.0 instead.